File: test-support/src/test/java/com/rafaelfiume/prosciutto/test/SalumeApiContractExampleReaderTest.java
package com.rafaelfiume.prosciutto.test;

import org.junit.Test;

import static com.rafaelfiume.prosciutto.test.SalumeApiContractExampleReader.supplierAdviceForExpertRequest;
import static com.rafaelfiume.prosciutto.test.SalumeApiContractExampleReader.supplierAdviceForExpertResponse;
import static org.hamcrest.Matchers.containsString;
import static org.junit.Assert.assertThat;

public class SalumeApiContractExampleReaderTest {

    //
    // Checking if request/response test setup is all right
    //

    @Test
    public void chekingRequestExampleIsNotNull() {
        assertThat(supplierAdviceForExpertRequest(), containsString("/salume/supplier/advise/for/"));
    }

    @Test
    public void chekingResponseExampleIsNotNull() {
        assertThat(supplierAdviceForExpertResponse(), containsString("<product-advisor>"));
    }

}

File: test-support/src/test/kotlin/com/rafaelfiume/prosciutto/test/SalumeApiContractExampleReaderTest.kt
package com.rafaelfiume.prosciutto.test

import org.junit.Test

import com.rafaelfiume.prosciutto.test.SalumeApiContractExampleReader.supplierAdviceForExpertRequest
import com.rafaelfiume.prosciutto.test.SalumeApiContractExampleReader.supplierAdviceForExpertResponse
import org.hamcrest.Matchers.containsString
import org.junit.Assert.assertThat

class SalumeApiContractExampleReaderTest {

    //
    // Checking if request/response test setup is all right
    //

    @Test
    fun checkingRequestExampleIsNotNull() {
        assertThat(supplierAdviceForExpertRequest(), containsString("/salume/supplier/advise/for/"))
    }

    @Test
    fun checkingResponseExampleIsNotNull() {
        assertThat(supplierAdviceForExpertResponse(), containsString("<product-advisor>"))
    }

}
